sale_stock_picking_blocking/tests/test_sale_stock_picking_blocking.py

```diff
@@ -28,15 +28,17 @@ def setUpClass(cls):
         # Create product:
         prod_dict = {
             "name": "test product",
-            "type": "product",
+            "type": "consu",
+            "categ_id": cls.env.ref("product.product_category_all").id,
+            "list_price": 100.0,
+            "standard_price": 60.0,
+            "uom_id": cls.env.ref("uom.product_uom_unit").id,
+            "uom_po_id": cls.env.ref("uom.product_uom_unit").id,
         }
-        product = cls.env["product.product"].with_user(cls.user_test).create(prod_dict)
+        cls.product = (
+            cls.env["product.product"].with_user(cls.user_test).create(prod_dict)
+        )
         # Create Sale order:
-        # TODO/TMP:
```

Comment:
You're removing the TODO, but the sale order is still created with a predefined name rather than it being assigned by a sequence. If we now know why this is necessary, can you document that here?